packages/drivekit_core/flutter_drivekit_core/example/lib/widgets/sliver_remove_all_listeners.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_drivekit_core/flutter_drivekit_core.dart';
+
+class SliverRemoveAllListeners extends StatelessWidget {
+  const SliverRemoveAllListeners({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SliverToBoxAdapter(
+      child: ElevatedButton(
+        onPressed: () {
+          DriveKitCore.instance.removeAllDriveKitListeners();
+        },
+        child: const Text('Remove all listeners'),
```

Comment:
```suggestion
        child: const Text('Remove all DriveKitListener'),
```

(Pour anticiper la présentation des `DKDeviceConfigurationListener` et `TripListener`